root/static/scripts/edit/components/ArtistCreditNameEditor.js

```diff
@@ -52,7 +52,7 @@ class ArtistCreditNameEditor extends React.Component {
     this.props.onChange({name: event.target.value});
   }
 
-  onJoinPhraseBlur(event) {
+  hanldeJoinPhraseBlur(event) {
```

Comment:
This looks like it might be the source of some or all of the test failures: "hanlde"

Reply:
Thank you so much for catching that!